Cabal/Cabal.cabal

```diff
@@ -410,6 +480,9 @@ library
     Distribution.Types.ComponentRequestedSpec
     Distribution.Types.TargetInfo
     Distribution.Utils.NubList
+    Distribution.Utils.UnionFind
+    Distribution.Utils.Base62
+    Distribution.Utils.Progress
```

Comment:
Just wondering about this D.Utils namespace. Is this really a public facing part of the Cabal API, or part of the build system, or part of the backpack subdivision of Cabal. (I'm distinguishing Cabal the declarative package specish bits from the "simple" build system.)

Do all of these need to be in exposed modules?